javascript/exercises/variables.html

```diff
@@ -15,18 +15,18 @@
 <h3 class="page-header">The Fortune Teller</h3>
 <p><em>Why pay a fortune teller when you can just program your fortune yourself?</em></p>
 <ul>
- <li>Store the following into variables: number of children, partner's name, geographic location, job title.
```

Comment:
oic, location is probably a reserved keyword in the language 😊 geoLocation is more descriptive than place I think :)